src/Libraries/Microsoft.Extensions.AI.Abstractions/Utilities/AIJsonUtilities.Schema.cs

```diff
@@ -46,39 +46,47 @@ public static partial class AIJsonUtilities
     private static readonly string[] _schemaKeywordsDisallowedByAIVendors = ["minLength", "maxLength", "pattern", "format"];
 
     /// <summary>
-    /// Determines a JSON schema for the provided AI function parameter metadata.
+    /// Determines a JSON schema for the provided method signature.
     /// </summary>
+    /// <param name="methodInfo">The method from which to extract schema information.</param>
     /// <param name="title">The title keyword used by the method schema.</param>
     /// <param name="description">The description keyword used by the method schema.</param>
-    /// <param name="parameters">The AI function parameter metadata.</param>
     /// <param name="serializerOptions">The options used to extract the schema from the specified type.</param>
     /// <param name="inferenceOptions">The options controlling schema inference.</param>
     /// <returns>A JSON schema document encoded as a <see cref="JsonElement"/>.</returns>
     public static JsonElement CreateFunctionJsonSchema(
+        MethodBase methodInfo,
```

Comment:
I'd recommend naming this simply `method`, because `methodInfo` sounds like it should be an instance of `MethodInfo`.

Reply:
Changed, thanks.